tests/integration/cv_classification.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+
+#include <opencv2/opencv.hpp>
+
+#include "cv/classification.hpp"
+#include "cv/segmentation.hpp"
+#include "protos/obc.pb.h"
+#include "utilities/constants.hpp"
+
+// target image paths, modify to use your targets
+const std::vector<std::string> targetPaths = {
+    "../tests/integration/images/target0.jpg",
+    "../tests/integration/images/target1.jpg",
+    "../tests/integration/images/target2.jpg",
+    "../tests/integration/images/target3.jpg",
+    "../tests/integration/images/target4.jpg"
+};
+
+// model weights path
+const std::string segmentationModelPath = "../models/fcn.pth";
+const std::string classificationShapeModelPath = "../models/thomas_shape_model.pt";
+const std::string classificationCharModelPath = "../models/thomas_char_model.pt";
+
+
+int main(int argc, char* argv[]) {
+    Segmentation segmentation(segmentationModelPath);
+    CroppedTarget ct;
+
+    Classification classification(classificationShapeModelPath, classificationCharModelPath, "");
+
+    for (int i = 0; i < targetPaths.size(); i++) {
+        auto target = cv::imread(targetPaths[i]);
+        ct.croppedImage = target;
+        auto result = segmentation.segment(ct);
+
+        cv::Mat characterMask(result.characterMask.size(), result.characterMask.type());
+        result.characterMask.copyTo(characterMask);
+        cv::Mat shapeMask(result.shapeMask.size(), result.shapeMask.type());
+        result.shapeMask.copyTo(shapeMask);
+
+        ClassificationResults results = classification.classify(target, shapeMask, characterMask);
+        std::string character = results.character;
+        std::string shape = results.shape;
+
+        std::cout << "For image " << targetPaths[i] << ": character=" << character \
+        << ", shape=" << shape << std::endl;
+
+        
+    }
```

Comment:
Really like this integration test. It's nice how it runs Segmentation first since that makes it easy for us to pass in generated image from not-stolen or real images to test.